absl/debugging/CMakeLists.txt

```diff
@@ -33,6 +35,8 @@ absl_cc_library(
     "stacktrace.cc"
   COPTS
     ${ABSL_DEFAULT_COPTS}
+  LINKOPTS
+    ${EXECINFO_LIBRARY}
```

Comment:
This fails when execinfo isn't available, maybe because it isn't needed.

```
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
EXECINFO_LIBRARY
    linked by target "stacktrace" in directory /abseil-cpp/absl/debugging

-- Generating done
CMake Generate step failed.  Build files cannot be regenerated correctly.
```

I think you need something like `$<$<BOOL:${EXECINFO_LIBRARY}>:${EXECINFO_LIBRARY}>`

Reply:
Indeed, I updated the PR